compiler/front_end/module_ir.py

```diff
@@ -793,9 +794,9 @@ def _type_definition(type_definition):
 def _structure(struct, name, parameters, colon, comment, newline, struct_body):
   """Composes the top-level IR for an Emboss structure."""
   del colon, comment, newline  # Unused.
-  struct_body.structure.source_location.start.CopyFrom(
+  ir_data_utils.builder(struct_body.structure).source_location.start.CopyFrom(
```

Comment:
With the current implementation of `builder` this has the same effect as before the change. But the comment on the `builder` implementation implies that this will change. Normally I'd expect something along the lines of

```
builder = ir_data_utils.builder(struct_body.structure) ...
builder.source_location.end ...
return builder.build()
```

I guess it seems odd that the return value appears to be ignored. Or will the future `builder()` impl still return something equivalent to `self`?

Reply:
We could maybe choose a different name, it's not exactly a builder in the "builder pattern" sense. The intention of the `builder` is to let you create a field by accessing it (the next patch will be adding a `reader` that lets you access a non-existent field, but it won't actually create it).